spec.html

```diff
@@ -452,24 +452,33 @@ <h1>Abstract Operations for Segmenter Objects</h1>
       A Segment Data Object is an object that represents a particular segment from a string.
     </p>
 
-    <emu-clause id="sec-findboundary" aoid="FindBoundary">
-      <h1>FindBoundary ( _segmenter_, _string_, _startIndex_, _direction_ )</h1>
-      <p>The FindBoundary abstract operation is called with arguments Intl.Segmenter instance _segmenter_, String value _string_, integer Number value _startIndex_, and _direction_ (which must be ~before~ or ~after~) to find a segmentation boundary between two code units in _string_ in the specified _direction_ from the code unit at index _startIndex_ according to the locale and options of _segmenter_ and return the immediately following code unit index (which will be infinite if no such boundary exists). The following steps are taken:</p>
+    <emu-clause id="sec-findboundary-before" aoid="FindBoundaryBefore">
+      <h1>FindBoundaryBefore ( _segmenter_, _string_, _startIndex_ )</h1>
+      <p>The FindBoundaryBefore abstract operation is called with arguments Intl.Segmenter instance _segmenter_, String value _string_, and integer Number value _startIndex_ to find a segmentation boundary between two code units in _string_ before the code unit at index _startIndex_ according to the locale and options of _segmenter_ and return the immediately following code unit index. The following steps are taken:</p>
+      <emu-note>Boundary determination is implementation-dependent, but general default algorithms are specified in Unicode Standard Annex 29 (available at <a href="https://www.unicode.org/reports/tr29/">https://www.unicode.org/reports/tr29/</a>). It is recommended that implementations use locale-sensitive tailorings such as those provided by the Common Locale Data Repository (available at <a href="http://cldr.unicode.org">http://cldr.unicode.org</a>).</emu-note>
```

Comment:
This is the duplication I don't want. Within ECMA 402, FindBoundary should be a single implementation-dependent operation with a suggestion to use CLDR tailorings of UAX 29, and it shouldn't imply that use of different boundary determination logic for forward vs. backward operations is acceptable.

Reply:
there are no such imply of "that use of different boundary determination" here. If you assume breaking into two operation would lead to such assumption, then putting them in two steps in the same operation could reach that very same assumption too, right?

Reply:
No, because when it's one operation the single "_Boundary determination is implementation-dependent..._" note covers both "Search _string_ for segmentation boundary" steps. It implies a tightness that I want to preserve, that there is in fact a single direction-parameterized FindBoundary operation rather than there being two fixed-direction FindBoundary operations.